### PR TITLE
API Key rotation in Portal

### PR DESCRIPTION
**Pull request by alex-yau-ttd**

### CODE REVIEW

File: docs/portal/api-keys.md [hunk 1]
```diff
@@ -85,8 +86,20 @@ Be sure that your implementation is updated with a new key before deleting an ac
 
 To delete an API key, follow these steps:
 
+1. In the UID2 Portal, go to the **API Keys** page.
```
Comment (alex-yau-ttd) [inline docs/portal/api-keys.md:89]: I added this line for consistency with the other sections.

**Review by genwhittTTD — APPROVED** [review]
LGTM thx!

File: docs/portal/api-keys.md [hunk 2]
```diff
@@ -85,8 +86,20 @@ Be sure that your implementation is updated with a new key before deleting an ac
...
+## Rotating an API key
+
+We recommend rotating each API key on a yearly basis. To rotate an API key, follow these steps:
+
+1. In the UID2 Portal, go to the **API Keys** page.
+1. Find the key in the list that needs to be rotated.
+1. Add a new key with the same permissions as the key that needs to be rotated. For details, see [Adding an API Key](#adding-an-api-key).
+1. Update your UID2 implementation to use the new key instead of the key that needs to be rotated.
+1. Ensure the new key is being used without issues.
+1. Delete the key that needs to be rotated. For details, see [Deleting an API Key](#deleting-an-api-key).
```
Comment (genwhittTTD) [inline docs/portal/api-keys.md:105]: Here you could say "the old key" -- but this is fine too.

**Review by genwhittTTD — COMMENTED** [review]
Left one minor update line 104.

File: docs/portal/api-keys.md [hunk 3]
```diff
@@ -85,8 +86,20 @@ Be sure that your implementation is updated with a new key before deleting an ac
...
+
+## Rotating an API key
+
+We recommend rotating each API key on a yearly basis. To rotate an API key, follow these steps:
+
+1. In the UID2 Portal, go to the **API Keys** page.
+1. Find the key in the list that needs to be rotated.
+1. Add a new key with the same permissions as the key that needs to be rotated. For details, see [Adding an API Key](#adding-an-api-key).
+1. Update your UID2 implementation to use the new key instead of the key that needs to be rotated.
+1. Ensure the new key is being used without issues, i.e. no degradation to the service or error logs related to the API key usage.
```
[the remark this answers is not in the document]
Comment (alex-yau-ttd) [reply docs/portal/api-keys.md:104]: thanks Gen - I've addressed that in https://github.com/IABTechLab/uid2docs/pull/642

**Pull request merged**